cpmonitor/templates/administration_checklist.html

```diff
@@ -45,7 +45,17 @@ <h2 class="accordion-header">
                  class="accordion-collapse collapse"
                  aria-labelledby="flush-heading{{ forloop.counter0 }}"
                  data-bs-parent="#accordionFlushKAP">
-              <div class="accordion-body">{{ item.help_text }}</div>
+              <div class="accordion-body">
+                <p>
+                  <strong>Erklärung:</strong>
+                </p>
+                <p>{{ values.help_text }}</p>
+                {% if values.rationale.strip %}
+                  <p>
+                    <strong>Anmerkung / Begründung:</strong> {{ values.rationale | safe_markdown }}
```

Comment:
Why is the rationale in the same `<p>` and not in a separate one, as the help text?